<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsForm</class>
 <widget class="QWidget" name="ContactsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/pixmaps/filter.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="categoryFilter">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="contactsFilter"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="contactView">
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addContactButton">
       <property name="text">
        <string>Aggiungi</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/pixmaps/user_add.png</normaloff>:/pixmaps/user_add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delContactButton">
       <property name="text">
        <string>Rimuovi</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/pixmaps/user_delete.png</normaloff>:/pixmaps/user_delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Modifica</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/pixmaps/user_edit.png</normaloff>:/pixmaps/user_edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionEditContact">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pixmaps/user_edit.png</normaloff>:/pixmaps/user_edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit Contact</string>
   </property>
  </action>
  <action name="actionHideColumn">
   <property name="text">
    <string>Hide Column</string>
   </property>
  </action>
  <action name="actionShowAllColumns">
   <property name="text">
    <string>Show All Columns</string>
   </property>
  </action>
  <action name="actionAddContact">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pixmaps/user_add.png</normaloff>:/pixmaps/user_add.png</iconset>
   </property>
   <property name="text">
    <string>Add Contact</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
